src/components/shared/TooltipHeaderWithDuration.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+
+import React, { PureComponent } from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  duration: number,
+  title: string,
+  className?: string,
+};
+
+export default class TooltipHeaderWithDuration extends PureComponent {
+  props: Props;
+
+  render(): React$Element<*> {
+    console.log('PLOP3');
+    let { duration } = this.props;
```

Comment:
I feel like it's about time to hoist [getFormattedTimeLength](https://github.com/devtools-html/perf.html/blob/5a6887d353c1909692c402d3b1b06d4038fb7015/src/profile-logic/range-filters.js#L28-L36) function up into `/src/utils/`. What do you think?